Week-03/css4.html

```diff
@@ -29,17 +42,18 @@
 </head>
 <body>
     <section class="blocks-container">
-        <div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div>
-        <div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div>
-        <div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div>
-        <div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div>
-        <div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div><div class="block">b</div>
+        <div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div><div class="yellow" class="block">b</div>
```

Comment:
I see your'e adding both `class="yellow"` and `class="block"` to this--nice! You don't actually have to add the `class` attribute twice, you can just do it once and separate the classes with a space: `class="yellow block"`